### PR TITLE
[test-framework](cloud) add necessary config and function for run cloud cases

### PR DESCRIPTION
**Pull request by pingchunzhang**
## Proposed changes

Issue Number: close #xxx

add necessary config and function in test framweork for run cloud cases

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by pingchunzhang** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/Config.groovy [hunk 1]
```diff
@@ -92,21 +99,51 @@ class Config {
...
+    public String clusterDir
+    public String kafkaBrokerList
+    public String cloudVersion
 
     Config() {}
 
     Config(String defaultDb, String jdbcUrl, String jdbcUser, String jdbcPassword,
-           String feSourceThriftAddress, String feTargetThriftAddress, String feSyncerUser, String feSyncerPassword,
-           String syncerPassword, String feHttpAddress, String feHttpUser, String feHttpPassword, String metaServiceHttpAddress,
+           String feSourceThriftAddress, String feTargetThriftAddress, String feSyncerUser, String feSyncerPassword, String syncerPassword,
```
Comment (gavinchou) [inline regression-test/framework/src/main/groovy/org/apache/doris/regression/Config.groovy:136]: make it a arg per line ?

**Comment by pingchunzhang** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**